tests/ClientFunctionalTest.php

```diff
@@ -208,6 +208,18 @@ public function testListSandboxesFromBranch(): void
             $createdSandbox->getId(),
             array_map(fn(Sandbox $s) => $s->getId(), $branchResponse)
         );
+
```

Comment:
I would expect this to be a separate test case :thinking:

Reply:
hmm, I thought that cleaning up the branch here fit quite nicely, so this test tests the full branch lifecycle

Reply:
then this test case should be named differently

Reply:
ah, sure, agreed, brb